ProjFS.Mac/PrjFSKextTests/MockVnodeAndMount.hpp

```diff
@@ -34,17 +45,16 @@ struct vnode
 public:
     bool isRecycling = false;
     vtype type = VREG;
+    VnodeMockErrors errors;
+    VnodeMockValues values;
```

Comment:
This PR is generally on the right track, but I've not merged it into what hopefully is a (near-) final version of my [PR #839](https://github.com/Microsoft/VFSForGit/pull/839) because I'm not convinced the `VnodeMockValues` struct is the right approach. I'm thinking more along the lines of having a public field `struct vnode_attr attrValues;` which is appropriately initialised by the constructors/`Create` functions.